src/api/authentication/AuthenticateJWT.ts

```diff
@@ -0,0 +1,32 @@
+import jwt from "jsonwebtoken";
+import { Request, NextFunction, Response } from "express";
+import { StatusCodes } from "http-status-codes";
+
+export const authenticateJWT = (
+  request: Request,
+  response: Response,
+  next: NextFunction,
+) => {
+  const secretKey = process.env.JWT_SECRET;
+  if (!secretKey) {
+    return {
+      statusCode: StatusCodes.BAD_REQUEST,
+      message: `secret key not provided`,
+      return: null,
+    };
+  }
```

Comment:
Se for pra arremessar um erro aqui, será que a culpa foi do cliente (status 4xx)?